internal/optimization_checks.py

```diff
@@ -304,6 +304,10 @@ def __init__(self, job, task, requests):
                         '.cdn.dnsv1.com.cn',
                         '.dsa.dnsv1.com',
                         '.dsa.dnsv1.com.cn'],
+            'Transparent Edge': ['.edge2befaster.io',
+                                 '.edge2befaster.net',
+                                 '.edgetcdn.io,
```

Comment:
Looks like you are missing a quote here

Reply:
You're completely right. I just added the change.